biz.aQute.bndlib/src/aQute/bnd/osgi/AnnotationHeaders.java

```diff
@@ -675,6 +676,26 @@ private String getFilter(Annotation a, Requirement annotation) {
 				}
 			}
 		}
+		else if (a.containsKey("version")) {
+			String floor = annotation.version();
+
+			int current = filter.lastIndexOf(")");
+
+			filter.append("(version>=")
+				.append(floor)
+				.append(")(!(version>=${version;+00;")
```

Comment:
`versionmask` is the preferred macro name to use. `version` is the old name and too close to the common variable name.